Makefile

```diff
@@ -1,23 +0,0 @@
-##########------------------------------------------------------------##########
```

Comment:
Any reason for removing the `Makefile` :stuck_out_tongue_closed_eyes:

Reply:
As veeresh is coming up with new model to test cli e2e, `Makefile` is no longer valid here